packages/main/src/SegmentedButton.ts

```diff
@@ -172,19 +199,20 @@ class SegmentedButton extends UI5Element {
 	}
 
 	normalizeSelection() {
-		const selectedItems = this.items.filter(item => item.pressed);
+		const selectedItems = this.pressedItems;
 		const selectedIndex = this._selectedItem ? selectedItems.indexOf(this._selectedItem) : -1;
 
 		if (this._selectedItem && selectedItems.length > 1) {
 			selectedItems.splice(selectedIndex, 1);
```

Comment:
- Isn't this logic also mode dependant, for example in Multi selection, should we still splice the selectedItems

- in general "selected" and "pressed" usage are mixed up although they mean the same thing in the context of SegmentedButton. I saw that you know use pressedItems for a public getter and public event param and it's now leading so to say. Then, start using it for variable names to make the code more readable and consistent:
const selectedItems = this.pressedItems; - > const pressedItems = this.pressedItems;
_selectedItem -> _pressedItem
selectedIndex -> pressedItemIndex

Reply:
Done